task_11/index.html

```diff
@@ -0,0 +1,351 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="Cache-Control" content="no-store">
+	<title>Order form</title>
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet">
+</head>
+<body>
+	<!-- Begin header section with nav and sign in -->
+	<header class="header">
+		<button class="header-button"><i class="fas fa-long-arrow-alt-left"></i>Go back</button>
+		<img src="images/logo.png" class="logo" alt="logo">
+		<button class="header-button"><i class="fas fa-user"></i>Sign in</button>
+	</header>
+	<!-- End header section with nav and sign in -->
+	<!-- Begin main section -->
+	<section class="order">
```

Comment:
I believe it can be `<main>` tag ?

Reply:
Thanks! Definitely it should be `<main>` here)